tests/integration/pure_cairo_vm/cairo1_integration/cairo1_runner_test.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
```

Comment:
More tests to add
- failing - probably in separate PR after we support it
- broken test suite (incorrect syntax)
- include other function and check if it is not collected by test collector

Reply:
https://github.com/software-mansion/protostar/issues/1550